app/services/idv/steps/in_person/state_id_step.rb

```diff
@@ -37,7 +35,9 @@ def call
            end
           end
 
-          maybe_redirect_to_verify_info(flow_session[steps[:address].to_s].blank?)
+          if flow_session['Idv::Steps::InPerson::AddressStep']
+            redirect_to idv_in_person_verify_info_url
+          end
```

Comment:
Why does this redirect? Is this when the user is updating their info?

Reply:
This does the same thing it did before this PR, just implemented differently. It lets the Flow State Machine control the flow unless someone has already completed the AddressStep, in which case they're editing and need to return to in_person verify_info.

Reply:
Thanks @soniaconnolly  I was pairing with another team member so did not get a chance to respond to both comments.